lib/drone/supermarket/config.rb

```diff
@@ -130,32 +76,36 @@ def default_logger
       #
       def write_keyfile
         keyfile_path.open "w" do |f|
-          f.write private_key
+          f.write payload[:private_key]
```

Comment:
If it's OK I'd like to avoid Hashie for now.  Never used it before and not sure how to use it with existing Hash provided by gli.